gc/base/standard/Scavenger.cpp

```diff
@@ -440,7 +438,6 @@ MM_Scavenger::workerSetupForGC(MM_EnvironmentStandard *env)
 	/* caches should all be reset */
 	Assert_MM_true(NULL == env->_survivorCopyScanCache);
 	Assert_MM_true(NULL == env->_tenureCopyScanCache);
-	Assert_MM_true(NULL == env->_scanCache);
```

Comment:
Is it any way to have a similar check with new logic?

Reply:
Not with the current changes.  We still verify that all of the copy caches are NULL.  Since there is technically not scanCache pointer in the env there is nothing to check.  I believe we still have an assert at the end of the GC to ensure that all caches are back on the empty list so I think we are covered.